### PR TITLE
NewRelic Lambda Integration Deployment Scripts

### PR DESCRIPTION
**Pull request by bowlofstew**
This pull request (PR) adds script support that I created while working thru the NewRelic deployment workflows.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/subhub/273)

### CODE REVIEW

File: bin/new-relic/enable-streaming.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    fi
+    if [ -d "$NR_LAMBDA_DIRECTORY" ]; then
+        cd nr-lambda
+        sh newrelic-cloud stream-lambda-logs $1
+    else
+        echo "Please run install.sh first."
+    fi
+}
+
+enable_streaming $1
```
Comment (sidler-mozilla) [inline bin/new-relic/enable-streaming.sh]: how will do this one or more?  shouldn't this be "$@"?

File: bin/new-relic/install.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+command -v git >/dev/null 2>&1 || {
```
Comment (sidler-mozilla) [inline bin/new-relic/install.sh]: `hash <program>` is a cleaner, better way to test if an executable is available.

Comment (bowlofstew) [reply bin/new-relic/install.sh]: Both are qualitative statements which I’d like to quantify here with pointing to the source code of both: https://github.com/bminor/bash/blob/master/builtins/command.def vs https://github.com/bminor/bash/blob/master/builtins/hash.def

Is this the case of the arg being used in cleaner?   Better: is the hashing of a previously traversed PATH location vs full PATH traversal on command.def?
I’m just trying to make the internal case for either/or here

File: bin/new-relic/enable-streaming.sh (hunk 1 above)
Comment (bowlofstew) [reply bin/new-relic/enable-streaming.sh]: Sure.

**Pull request merged**